assets/queries/ansible/aws/cloudformation_stack_notifications/test/positive_expected_result.json

```diff
@@ -0,0 +1,7 @@
+[
+	{
+		"queryName": "AWS CloudFormation Stack Notifications",
```

Comment:
Replace the name with "AWS CloudFormation Stack Notifications Disabled", to be easier to understand what the vulnerability is.